modules/microshift-firewall-update-for-service.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-firewall.adoc
+
+:_content-type: CONCEPT
+[id="microshift-firewall-update-for-service_{context}"]
+= Updating firewall ports when a service is exposed
+
+Firewalld is often active when you run services on a {product-title}. This can disrupt certain services on a {product-title} because traffic to the ports might be blocked by the firewall. You need to ensure the necessary firewall ports are open if you want certain services to be accessible from outside the host. There are several options for opening your ports:
+
+* Services of the `NodePort` and `LoadBalancer` type are automatically available with OVN-K.
++
+In these cases, OVN-K adds iptables rules so the traffic to the cluster IP address is delivered to the relevant ports. This is done using the PREROUTING rule chain and is then forwarded to the OVN-K to bypass the firewalld rules for local host ports and services. Iptables and firewalld are backed by nftables in RHEL9. The nftables rules, that the iptables generates, always has priority over the rules that the firewalld generates.
+
+* Pods with the `HostPort` parameter settings are automatically available. This also includes the router-default pod, which uses ports 80 and 443.
++
+For `HostPort` pods, the CRI-O config sets up DNAT to the pods IP address and port. The CRI-O config sets up the pods with the iptables rules as well.
```

Comment:
For `HostPort` pods, the CRI-O config sets up DNAT iptables rules to the pods IP address and port.

Reply:
Oh gotcha, my misunderstanding here, thank you!